<ui version="4.0" >
 <class>gameLobbyDialog</class>
 <widget class="QDialog" name="gameLobbyDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Game Lobby</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="treeWidget_GameList" >
     <column>
      <property name="text" >
       <string>Game</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Lobby-Chat</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>7</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTextBrowser" name="textBrowser_ChatDisplay" />
      </item>
      <item row="1" column="0" >
       <widget class="QLineEdit" name="lineEdit_ChatInput" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>181</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CreateGame" >
       <property name="text" >
        <string>Create Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_JoinGame" >
       <property name="text" >
        <string>Join Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeWidget_GameList</tabstop>
  <tabstop>pushButton_CreateGame</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>gameLobbyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>465</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CreateGame</sender>
   <signal>clicked()</signal>
   <receiver>gameLobbyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel" >
     <x>125</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
